etcdserver/raft.go

```diff
@@ -226,6 +226,10 @@ func (r *raftNode) start(s *EtcdServer) {
 				if err := r.storage.Save(rd.HardState, rd.Entries); err != nil {
 					plog.Fatalf("raft save state and entries error: %v", err)
 				}
+				if !raft.IsEmptyHardState(rd.HardState) {
+					proposalsCommitted.Set(float64(rd.HardState.Commit))
```

Comment:
If you find yourself doing this, it hints this metric isn't being updated in the right place.

In this case as you're talking about the absolute values being useful, this is probably actually a gauge. For a counter you only really care about it's rate of increase.